src/app-logic/browser-connection.js

```diff
@@ -97,6 +102,20 @@ class BrowserConnectionImpl implements BrowserConnection {
     return profile;
   }
 
+  async querySymbolicationApi(
+    path: string,
+    requestJson: string
+  ): Promise<string> {
+    // This only works on Firefox 96 and above.
+    if (!this._webChannelSupportsGetProfileAndSymbolication) {
+      throw new Error(
+        "Can't use querySymbolicationApi in Firefox versions with the old WebChannel."
+      );
+    }
+
+    return querySymbolicationApiViaWebChannel(path, requestJson);
```

Comment:
It's useful so that we can add more APIs without having to add more methods to the WebChannel. I want to add two new APIs: A way to get source code, and a way to get assembly code.
I've already landed a `/source/v1` API in profiler-get-symbols (but not integrated it into Firefox yet): https://github.com/mstange/profiler-get-symbols/issues/24